### ISSUE TITLE
Make the lib compatible with Google Translate

### ISSUE THREAD

**Issue by zallek** [connor-baer/rich-text-to-jsx#41]
## Purpose

Make the lib compatible with Google Translate

When Google Translate a web page, it replaces text nodes with `<font>` tags containing translations while React keeps references to the text nodes that are no longer in the DOM tree. Leading to React error and a blank page.

More info [here](https://lifesaver.codes/answer/make-react-resilient-to-dom-mutations-from-google-translate) 

## Approach & Changes

When facing the following paragraph
```JSON
{
  "nodeType": "paragraph",
  "content": [
    {
      "nodeType": "text",
      "value": "Welcome",
      "marks": [],
      "data": {}
    },
    {
      "nodeType": "hyperlink",
      "content": [
        {
          "nodeType": "text",
          "value": "Something",
          "marks": [],
          "data": {}
        }
      ],
      "data": {
        "uri": "foo"
      }
    }
  ],
  "data": {}
}

```

The lib currently renders the following. Therefore Google Translate will replace `Welcome` by `<font>Welcome</font>` leading to a React error on the next update.

```jsx
<p>
  Welcome
  <a>Something</a>
</p>
```

As shown in this [article](https://lifesaver.codes/answer/make-react-resilient-to-dom-mutations-from-google-translate), to solve the issue, we should render the following so that Google Transform will replace `<span>Welcome</span>` by `<span>Bienvenue</span>` preventing bugs as the DOM tree isn't changed.

```jsx
<p>
  <span>Welcome</span>
  <a>Something</a>
</p>
```

The current behavior is due to this [line](https://github.com/connor-baer/rich-text-to-jsx/blob/ad292fea86aa91ba303265f61302b0fcc12d55ff/src/rich-text-to-jsx.js#L119).
It would be great if we could override the text render when no marks.

Thanks.

**Comment by zallek** [comment]
Thanks for the quick reply.

Sure. I'll push a PR.

**Comment by connor-baer** [comment]
Thank you! 🙌

**Comment by connor-baer** [comment]
Closed by #42 and released in v2.2.0:

- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@madebyconnor/rich-text-to-jsx/v/2.2.0)
- [GitHub release](https://github.com/connor-baer/rich-text-to-jsx/releases/tag/2.2.0)
